creator-node/src/services/sync/syncImmediateQueue.js

```diff
@@ -96,22 +96,27 @@ class SyncImmediateQueue {
       creatorNodeEndpoint,
       forceResyncConfig,
       logContext,
-      serviceRegistry
+      serviceRegistry,
+      syncType,
+      syncUuid
     } = job.data
 
     const startTime = getStartTime()
     try {
-      await secondarySyncFromPrimary({
+      const result = await secondarySyncFromPrimary({
         serviceRegistry,
         wallet,
         creatorNodeEndpoint,
         forceResyncConfig,
-        logContext
+        logContext,
+        syncType,
+        syncUuid: syncUuid || null
       })
       logInfoWithDuration(
         { logger, startTime },
         `syncImmediateQueue - secondarySyncFromPrimary Success for wallet ${wallet} from primary ${creatorNodeEndpoint}`
       )
+      return result
```

Comment:
so this will basically now set the result in the bull dashboard right
i remember @dmanjunath had intentionally not done this earlier but idr why, adding this makes sense to me

Reply:
this doesn't short circuit the throw, so it should still add to failed when needed. the reason I added it was because I wanted to see the Bull result on the processing side to confirm that it matches the result that the issuing side now receives

Reply:
makes sense to me